ddprof-lib/src/main/cpp/rustDemangler.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021, 2023 Datadog, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <string>
+
+namespace RustDemangler {
```

Comment:
Oh, namespace :)
First time I see it here. It's been a long time, I almost forgot you could do this in C++ 🤣

Reply:
Heh, mostly here I was afraid of conflicting with something else in the project (I'm not familiar with the codebase).  Happy to remove if it would align with the design of the repo.  :pray:

Reply:
I don't think is necessary to remove it - I assume it is a purely syntactic construct and won't result in linking more of libstdc++ to the final library ...